routes/articles.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const router = express.Router();
+
+const articlesData = require('../database/articles.js');
+const articlesArray = articlesData.getArticlesArray();
+const articlesObject = articlesData.getArticlesObject();
+
+//
+router
+  .route('/')
+  .get((req, res) => {
+    if (articlesObject.messageCheck === false) {
+      articlesObject.message = '';
+    } else {
+      articlesObject.messageCheck = false;
+    }
+    res.status(200);
+    return res.render('./templates/articles/index', articlesObject);
+  })
+  .post((req, res) => {
+    let body = req.body;
+    articlesData.postArticle(body);
+    return res.render('./templates/articles/index', articlesObject);
+  });
+
+// /new
+
+router.get('/new', (req, res) => {
+  res.status(200);
```

Comment:
You don't need this line, since a render will overwrite the status to 200 anyway.